ui/src/onboarding/components/LoginForm.tsx

```diff
@@ -87,12 +91,23 @@ export const LoginForm: FC<Props> = ({
           </Grid.Column>
         </Grid.Row>
       </Grid>
-      <a onClick={handleForgotPasswordClick} className="login--forgot-password">
-        Forgot Password?
-      </a>
+      {
+        // @ts-ignore
+        // TODO: remove this ts-ignore when clockface v2.0.2 is released
+        <Heading
```

Comment:
I added this `@ts-ignore` here since I was getting a type issue when adding an `onClick` event to the `Heading` Component since the functionality doesn't exist on `clockface v2.0.1`. This PR added that functionality onto the Heading component:

https://github.com/influxdata/clockface/pull/449

but a new version of clockface has yet to be released